robot/Cumulus/resources/NPSPSettingsPageObject.py

```diff
@@ -29,6 +29,7 @@ def open_main_menu(self,title):
         locator=npsp_lex_locators["npsp_settings"]["main_menu"].format(title)
         self.selenium.wait_until_page_contains_element(locator, 
                                                error=f"click on {title} link was not successful even after 30 seconds")
+        self.selenium.capture_page_screenshot()                                        
```

Comment:
Just curious does the tag @capture_screenshot_on_error not help here ?

Reply:
no bcoz its not failing on this keyword, test fails 2 steps after this so curious on how this is passing